real_estate_module/models/estate_property_offer.py

```diff
@@ -0,0 +1,85 @@
+from odoo import api, fields, models
+from datetime import timedelta
+from odoo.exceptions import UserError
+
+class EstatePropertyOffer(models.Model):
```

Comment:
```suggestion
from odoo.exceptions import UserError


class EstatePropertyOffer(models.Model):
```
Exactly two empty line before defining a class